lib/usd/translators/shading/mtlxStandardSurfaceReader.cpp

```diff
@@ -13,55 +13,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-#include "mtlxBaseReader.h"
+#include "mtlxTranslationTableReader.h"
```

Comment:
There was a lot of common code between StandardSurface ad OpenPBRSurface converters. These commonalities were extracted in a base class.

Reply:
Great thank you.